frontend/packages/knative-plugin/src/plugin.tsx

```diff
@@ -382,6 +411,21 @@ const plugin: Plugin<ConsumedExtensions> = [
       hide: hideDynamicEventSourceCard,
     },
   },
+  {
+    type: 'AddAction',
+    flags: {
+      required: [FLAG_KNATIVE_EVENTING],
+    },
+    properties: {
+      id: 'knative-eventing-channel',
+      url: '/channel',
+      label: 'Channel',
+      description:
+        'Create a Channel to create an event forwarding and persistence layer with in-memory and reliable implementations',
+      icon: <AutomationIcon />,
+      hide: hideDynamicEventSourceCard,
```

Comment:
should this be new util `hideDynamicChannelCard` dependent on channel CRDs check?

Reply:
Yes, updated